lib/matplotlib/style/core.py

```diff
@@ -34,11 +34,27 @@
 STYLE_FILE_PATTERN = re.compile('([\S]+).%s$' % STYLE_EXTENSION)
 
 
+# A list of rcParams that should not be applied from styles
+STYLE_BLACKLIST = set([
+    'interactive', 'backend', 'backend.qt4', 'webagg.port',
+    'webagg.port_retries', 'webagg.open_in_browser', 'backend_fallback',
+    'toolbar', 'timezone', 'datapath', 'figure.max_open_warning',
+    'savefig.directory', 'tk.window_focus', 'hardcopy.docstring'])
+
+
+def _blacklist_style_params(d):
```

Comment:
The name is not so clear in indicating that a dict of valid params will be returned; how about `_screened_style_params` or `_valid_style_params`?